frontend/app/src/components/graphs/AssetAmountAndValueDialog.vue

```diff
@@ -0,0 +1,86 @@
+<script setup lang="ts">
```

Comment:
Do we use this component anywhere else in the app? If not this should not be the right place for it.